nil/internal/collate/proposer.go

```diff
@@ -351,10 +351,6 @@ func (p *proposer) handleTransactionsFromPool() error {
 	return nil
 }
 
-func (p *proposer) isTxProcessed(dbTx db.RoTx, txHash common.Hash) (bool, error) {
```

Comment:
Is it possible to drop collator state? Isn't 
it actually  TxCount inside block?